### ISSUE TITLE
"Filtering and streamcopy cannot be used together"

### ISSUE THREAD

**Issue by kocane** [mdhiggins/sickbeard_mp4_automator#848]
I am trying to use this excellent tool to force re-encode files on demand (using the manual.py script). 

I've set a `video-crf = 22` and audio should just be copied so I set `audio-codec = copy` and `audio-copy-original = True` (this shouldn't result in duplicates or anything, right?). In `postopts `I've set the following: `-preset,medium,-tune,film,-vf,scale=-2:720`. 

Now, when I run the script it gives me:
```
Filtergraph 'scale=-2:720' was defined for video output stream 0:0 but codec copy was selected.
Filtering and streamcopy cannot be used together.
```

Naturally it's because `video-codec` is set to `h264,x264` which is what I want, so it just copies that. But sholdn't this be overruled by `convert-mp4` which I've set to `True`?

In short: how can I force the script to _always_ convert to x264 with the settings I've set here? I gotta avoid it copying the vcodec, so I can use filtering, no?
